### PR TITLE
Removed Github Constellation link

### LINKED ISSUE #23: Remove broken github constellation banner
the github constellation page is constantly updating, so the below link is now broken
https://github.com/ohcnetwork/landing/blob/02da173f725887780f6df6ca9f977b9a31a6b6ac/pages/index.js#L149-L152

### PR DESCRIPTION
**Pull request by nithish1018**
Closes #23

### CODE REVIEW

File: pages/index.js
Comment (sainak) [inline pages/index.js:1]: setting banner to null should solve the issue without the need for removing the html

Comment (nithish1018) [reply pages/index.js:1]: ok, I'll revert the changes

**Review by sainak — APPROVED** [review]

**Review by bodhish — CHANGES_REQUESTED** [review]
Would be nice if you can document the shape of the state so that next time its easier to add.

**Comment by shivankacker** [comment]
@nithish1018 do update the PR as per changes requested

**Comment by nithish1018** [comment]
@bodhish @shivankacker I should just add comments for that **banner** state?.
Can I add the following comments?:
// `banner` state
// - Purpose: Stores information about the banner to display (URL for the Github constellation Page)
// - Initial value: null (no banner is set initially)
// - Expected type: string (URL of the banner image)

**Comment by shivankacker** [comment]
@nithish1018 Just create a new type `Banner` and make the state adhere to it or be null, just so we know the types.

File: pages/index.tsx [hunk 1]
```diff
@@ -827,7 +826,7 @@ export default function Home() {
             </div>
           </div>
         </section>
-        <CommitLayout />
+        <CommitLayout>{null}</CommitLayout>
```
Comment (rithviknishad) [inline pages/index.tsx:829]: is this change necessary?

File: tsconfig.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     ".next/types/**/*.ts",
     "**/*.ts",
     "**/*.tsx"
-  ],
+, "pages/index.tsx"  ],
```
Comment (rithviknishad) [inline tsconfig.json:32]: why was this changed?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (nithish1018) [reply tsconfig.json:32]: That updated automatically when I renamed index.js to index.tsx

Comment (rithviknishad) [reply tsconfig.json:32]: remove it? `**/*.tsx` covers it anyways.

Comment (nithish1018) [reply tsconfig.json]: Done

**Review by shivankacker — APPROVED** [review]

**Pull request merged**